deploy/cloudformation/manifest-pipeline.yml

```diff
@@ -686,3 +687,62 @@ Resources:
               finalizeLambdaArn: !GetAtt [ FinalizeLambdaFunction, Arn ],
             }
       RoleArn: !GetAtt [ StatesExecutionRole, Arn ]
+
+  DebugPermissions:
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      Roles:
+        - !ImportValue
+            Fn::Sub: ${InfrastructureStackName}:DebugRoleName
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Sid: AllowReadBuckets
+            Effect: Allow
+            Action:
+              - 's3:GetObject'
+              - 's3:ListBucket'
+            Resource:
+              - !GetAtt ProcessBucket.Arn
+              - !Sub ${ProcessBucket.Arn}/*
+              - !GetAtt ManifestBucket.Arn
+              - !Sub ${ManifestBucket.Arn}/*
+          - Sid: AllowStateMachineActions
+            Effect: Allow
+            Action:
+              - 'states:DescribeStateMachine'
+              - 'states:ListExecutions'
+              - 'states:StartExecution'
+              - 'states:DescribeExecution'
+              - 'states:GetExecutionHistory'
+              - 'states:DescribeStateMachineForExecution'
+            Resource:
+              - !Sub 'arn:aws:states:${AWS::Region}:${AWS::AccountId}:stateMachine:${StateMachine.Name}'
+              - !Sub 'arn:aws:states:${AWS::Region}:${AWS::AccountId}:execution:${StateMachine.Name}:*'
+
+  DataAdminPermissions:
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      Roles:
+        - !ImportValue
+            Fn::Sub: ${InfrastructureStackName}:DataAdminRoleName
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Sid: AllowReadBuckets
+            Effect: Allow
+            Action:
+              - 's3:GetObject'
+              - 's3:ListBucket'
+            Resource:
+              - !GetAtt ProcessBucket.Arn
+              - !Sub ${ProcessBucket.Arn}/*
+              - !GetAtt ManifestBucket.Arn
+              - !Sub ${ManifestBucket.Arn}/*
+          - Sid: AllowWriteToManifests
+            Effect: Allow
+            Action:
```

Comment:
Does the `DataAdmin` need to make changes to the data in the Process bucket directly?

Reply:
We did discuss this a bit, but we determined that this may conflict with what's being done with the storage gateway. I believe @SteveMattison was working on an alternate approach for how the data flows between the buckets to avoid issues, so we're holding off on adding permissions until that gets sorted out.

Reply:
Per @SteveMattison they will not need permissions to write to the process bucket. They will make use of the storage gateway attached to the process bucket as much as possible.